com.cuperino.qprompt.json

```diff
@@ -0,0 +1,40 @@
+{
+   "command": "qprompt",
+   "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--filesystem=host",
+      "--share=network",
```

Comment:
Does it really need both host and network access?

Reply:
--filesystem=host is necessary because users need to be able to read and write documents with QPrompt. These documents can be in plain text, HTML, and markdown formats. Future versions will also include read only support for PDF.

--share=network is only necessary to be able to load avatars from the about page's list of contributors. Users could disable this functionality without any implications. Nevertheless, I'd rather have networking on because, in some systems, failing to download the avatar results in QPrompt crashing. This is likely an upstream bug with KDE's Kirigami framework.

Reply:
Removing --share=network does not crash QPrompt while using the 5.15 runtime. I think it can be safely removed.

Reply:
Ideally, it should use a portal, which would let you drop any filesystem permissions. See https://docs.flatpak.org/en/latest/portals-qt.html

Reply:
@barthalion Thanks for letting me know. It helped me realize the portals where already in use, thus filesystem=host permissions are not necessary.

I also removed network permissions because the ability to download avatar pictures is non-essential to QPrompt, and its implementation has to be changed to use portals upstream, at the Kirigami framework.